app/components/Admin/NaicsCodeList.tsx

```diff
@@ -0,0 +1,30 @@
+import Link from 'next/link';
+import {useRouter} from 'next/router';
+import React from 'react';
+import {ListGroup} from 'react-bootstrap';
+
+interface Props {
+  naicsCodes: {code: string; description: string; id: string}[];
+}
+
+export const NaicsCodeList: React.FunctionComponent<Props> = ({naicsCodes}) => {
+  const router = useRouter();
+
+  return (
+    <ListGroup variant="flush">
+      {naicsCodes.map(({code, description, id}) => (
+        <Link
+          key={id}
+          prefetch={false}
+          href={{pathname: router.pathname, query: {naicsCodeId: id}}}
+        >
+          <ListGroup.Item active={router.query?.naicsCodeId === id}>
```

Comment:
Adding the `action` prop to `ListGroup.Item` ([docs example](https://react-bootstrap.github.io/components/list-group/#actionable-items)) improves the accessibility:
- makes the items render as `<button>` which enables interaction with them via keyboard focus navigation
- adds background hover style and `cursor: pointer;` to make it very clear that the list items are meant to be interacted with.